### PR TITLE
PR: Milestone 2

### PR DESCRIPTION
**Pull request by alexawesomecode**
Hello

I worked on all the requirements:

- Created components folder for App, Button, Display and ButtonDisplay components
- Made all of them without state into functional stateless components, and one into a Component Class
- Used best practices: destructuring objects, naming conventions, etc.

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import Button from './Button';
+import ButtonPanel from './ButtonPanel';
+
+
+function App() {
+  return (
+    <div>
+
+      <Display value="OOO" />
+      <Button name="boton" color="white" wide />
```
Comment (bellom) [inline src/components/App.js]: Let's make sure to keep the project going without no confusion 😄 , you don't need to add `Button` component in here cause it won't be in here in the later milestone.
kindly remove 👍 
image

File: src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = props => {
+  const { color, name } = props;
+
+  return (
+    <div className="button" style={{ backgroundColor: color }}>
```
Comment (bellom) [inline src/components/Button.js]: Your styling should be in CSS file, but you might wanna use a ternary operator later in the milestone.

File: src/components/Display.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = props => {
+  const { value } = props;
```
Comment (bellom) [inline src/components/Display.js]: ### [Optional]
To make the code more clear, you might wanna rename `value` to `result`
image

File: src/components/Button.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+      </h2>
+    </div>
+  );
+};
+
+
+Button.propTypes = {
+
+  name: PropTypes.string.isRequired,
+  color: PropTypes.string.isRequired,
```
Comment (bellom) [inline src/components/Button.js]: You're really moving fast which is nice 👍 
I guess `color` would be a `boolean` in later milestone if thou it not required in this milestone.

File: src/components/ButtonPanel.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+      <Button name="percent" />
+      <Button name="division" />
+    </div>
+
+    <div className="group-2">
+      <Button name="7" />
+      <Button name="8" />
+      <Button name="9" />
+      <Button name="X" />
+
```
Comment (bellom) [inline src/components/ButtonPanel.js]: You might wanna remove this line, makes your code scattered and also practice good indentation in the rest of the code 👍

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @alexawesomecode  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly check out my comments below;**

**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: src/components/ButtonPanel.js (hunk 5 above)
Comment (alexawesomecode) [reply src/components/ButtonPanel.js]: Thanks for the detailed comments! This was very helpful. Will  work on it :)

**Review by shloch — APPROVED** [review]
## good to go................on to the next milestone, we go
accomplished

**Pull request merged**